pkg/webhook/webhook.go

```diff
@@ -98,9 +98,6 @@ func (pwh *pravegaWebhookHandler) mutatePravegaVersion(ctx context.Context, p *p
 	if p.Spec.Pravega != nil && p.Spec.Pravega.Image != nil && p.Spec.Pravega.Image.Tag != "" {
 		p.Spec.Pravega.Image.Tag = ""
 	}
-	if p.Spec.Bookkeeper != nil && p.Spec.Bookkeeper.Image != nil && p.Spec.Bookkeeper.Image.Tag != "" {
```

Comment:
I guess we need to find a new way to ensure that Pravega and Bookkeeper are using the compatible versions.

Reply:
That would be managed using an uber helm chart that will deploy all pravega dependencies and components. See: https://github.com/pravega/pravega-operator/issues/294